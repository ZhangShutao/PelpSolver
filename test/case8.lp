innocent(X) :- K(0,1] not guilty(X), suspect(X).
warrant(X) :- K[0.5, 1] guilty(X), suspect(X).

guilty(X) :- witness(X), suspect(X).
guilty(X) :- not alibi(X), suspect(X). [1]
guilty(X) :- motivation(X), suspect(X). [1]
guilty(X) | not guilty(X) :- suspect(X).

suspect(tom).
motivation(tom).