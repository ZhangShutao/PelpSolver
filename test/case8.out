WorldView 1:
{[0.11920292202211755]:{innocent(tom),suspect(tom),warrant(tom),motivation(tom)},[0.8807970779778824]:{innocent(tom),guilty(tom),suspect(tom),warrant(tom),motivation(tom)}}